.github/workflows/tests.yml

```diff
@@ -24,3 +27,67 @@ jobs:
           && !contains(github.event.pull_request.labels.*.name, 'ci:run-full-test-suite')
           || false
          }}
+
+  upload-stable-deps:
+    needs: tests
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' }}

```

Comment:
Is this the line that ensures that we only create a PR on pushes to `master` and not other branches?